### ISSUE TITLE
Double entries when changing dir

### ISSUE THREAD

**Issue by ahmedkhalf** [kyazdani42/nvim-tree.lua#586]
Reproduction:
1. Use the following config:
```lua
    vim.g.nvim_tree_update_cwd = 1
    vim.g.nvim_tree_respect_buf_cwd = 1
```
2. Open nvim with `nvim -c "NvimTreeOpen"`
3. Entries are doubled.

![image](https://user-images.githubusercontent.com/36672196/130475218-119acf59-cc6e-4152-905b-3c9557071792.png)

**Comment by kaykhan** [comment]
@ahmedkhalf How did you fix this issue ? https://github.com/kyazdani42/nvim-tree.lua/issues/584

**Comment by AllenDang** [comment]
Same issue here.

**Comment by kyazdani42** [comment]
fixed in  0a178d17bc7a1c56cc

**Comment by ahmedkhalf** [comment]
thank you! fixed.
